opal-core-ws/src/main/java/org/obiba/opal/web/analysis/TableAnalysisResource.java

```diff
@@ -0,0 +1,124 @@
+package org.obiba.opal.web.analysis;
+
+import java.io.BufferedOutputStream;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import javax.activation.MimetypesFileTypeMap;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+
+import org.obiba.opal.core.service.AnalysisExportService;
+import org.obiba.opal.core.service.OpalAnalysisResultService;
+import org.obiba.opal.core.service.OpalAnalysisService;
+import org.obiba.opal.web.model.Projects;
+import org.obiba.opal.web.model.Projects.OpalAnalysisDto;
+import org.obiba.opal.web.model.Projects.OpalAnalysisDto.Builder;
+import org.obiba.opal.web.model.Projects.OpalAnalysisResultDto;
+import org.obiba.opal.web.model.Projects.OpalAnalysisResultsDto;
+import org.obiba.opal.web.project.Dtos;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class TableAnalysisResource {
+
+  private final OpalAnalysisService analysisService;
+  private final OpalAnalysisResultService analysisResultService;
+  private final AnalysisExportService analysisExportService;
+
+  private String datasourceName;
+  private String tableName;
+
+  @Autowired
+  public TableAnalysisResource(OpalAnalysisService analysisService,
+                                OpalAnalysisResultService analysisResultService,
+                                AnalysisExportService analysisExportService) {
+
+    this.analysisService = analysisService;
+    this.analysisResultService = analysisResultService;
+    this.analysisExportService = analysisExportService;
+  }
+
+  @GET
+  @Path("analyses")
+  public Projects.OpalAnalysesDto getProjectTableAnalyses(@QueryParam("order") @DefaultValue("desc") String order,
+                                                          @QueryParam("limit") @DefaultValue("0") int limit) {
```

Comment:
What's the use of this ``limit``: you cannot page the analyses list because there is no ``from`` and the DTO does not report the ``total count``.